Pa3/Durchführung/Project/src/app/State/DriveToFinish.c

```diff
@@ -15,6 +15,7 @@
 #include "service/LineSensor.h"
 
 /* CONSTANTS **************************************************************************************/
+#define TRESHOLD_LINE_RECOGNIZED 500
```

Comment:
codingstyle wants brackets

DriveOverGap uses the same Threshold. We should use something global:
CalibrateLineSensors.h:
 #define CALIB_NO_LINE(value) ((value) <= 500)
#define CALIB_OVER_LINE(value) ((value) > 500)